app/uk/gov/hmrc/gform/sharedmodel/formtemplate/AuthConfig.scala

```diff
@@ -135,6 +141,16 @@ object HasEnrolmentSection {
     }
 }
 
+object HasEmailConfirmation {
+  def unapply(ac: AuthConfig): Option[LocalisedString] =
+    ac match {
+      case EmailAuthConfig(_, _, _, Some(emailConfirmation)) => Some(emailConfirmation)
+      case Composite(configs) if configs.exists(_.isEmailConfirmation) =>
```

Comment:
I think this `if configs.exists(_.isEmailConfirmation)` is not needed now and so `def isEmailConfirmation` can also be removed?